.changeset/witty-squids-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
```

Comment:
Probably just wanna change this changeset a little bit to be more reflective of the actual change now.